### PR TITLE
Python38 as default

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/SeldonIO/seldon-core/issues/3500

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: doc/source/reference/images.md
Comment (agrski) [inline doc/source/reference/images.md:27]: :question: Should this be updated to have `1.10.0` as the stable version, given that's listed as the stable for `Seldon Python 3` above?
I'd also expect it to be stable given it's the default.

```suggestion
| [Seldon Python 3.8 Wrapper for S2I](../python/python_wrapping_s2i.md) | [seldonio/seldon-core-s2i-python38](https://hub.docker.com/r/seldonio/seldon-core-s2i-python38/tags/) |  1.10.0  | 1.11.0-dev |
```

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 7a91e0e94a6de472ec93858524e2a8a947b9e860 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-75jwk) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
One integration failure on `test_rolling_deployment[graph1.json-graph2.json-True-ambas]`
/test integration

File: doc/source/reference/images.md
Comment (agrski) [inline doc/source/reference/images.md:30]: Same question here about stable version - sorry, missed it first time round

Comment (RafalSkolasinski) [reply doc/source/reference/images.md:30]: In 1.10 it was new so there was no stable version and now there is.

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | b7f458565aa17e6baae558fe1ff2ad3e6a44c25a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-ccgvs) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | b7f458565aa17e6baae558fe1ff2ad3e6a44c25a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-ccgvs) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Integration failures on:
- `TestPrepack.test_xgboost_v2`
- `TestPrepack.test_mlflow_v2`

**Comment by seldondev** [comment]
@RafalSkolasinski: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | b7f458565aa17e6baae558fe1ff2ad3e6a44c25a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-ccgvs) | `/test integration`
notebooks | 7a91e0e94a6de472ec93858524e2a8a947b9e860 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-3591-notebooks-rwlcn) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Notebook failure:
- `TestNotebooks.test_explainer_examples`

**Comment by seldondev** [comment]
@RafalSkolasinski: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 7a91e0e94a6de472ec93858524e2a8a947b9e860 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-3591-notebooks-rwlcn) | `/test notebooks`
integration | a996949547f6affd108bd20fbbc8cc3d3eb3512b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-fl5mq) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by agrski — APPROVED** [review]
LGTM

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[agrski](https://github.com/SeldonIO/seldon-core/pull/3591#pullrequestreview-755311364 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@RafalSkolasinski: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 7a91e0e94a6de472ec93858524e2a8a947b9e860 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-3591-notebooks-rwlcn) | `/test notebooks`
integration | a996949547f6affd108bd20fbbc8cc3d3eb3512b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-dw868) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
we do have successful notebook pass 
image

**Comment by RafalSkolasinski** [comment]
integration failures on:
- `test_rolling_deployment[graph1.json-graph4.json-True-ambas]`
- `TestPrepack.test_mlflow_v2`
- `TestPrepack.test_xgboost_v2`

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | a996949547f6affd108bd20fbbc8cc3d3eb3512b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-2q86h) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | a996949547f6affd108bd20fbbc8cc3d3eb3512b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-4gvjm) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Verified locally prepack tests. They pass without issues and further on they cannot be broken by this change as these servers are based on `...-ubi8` images with specify explicitly python versions with `...-python37-ubi8`.
image

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | a996949547f6affd108bd20fbbc8cc3d3eb3512b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3591-integration-kmzjl) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Tested the rolling updates locally, everything green:
image

Also worth to mention that `test_rolling_deployment[graph1.json-graph4.json-True-ambas]` is testing `seldonio/fixed-model:0.1`  update from engine to executor deployment and the fixed model image is build with:
```
	s2i build -E environment_v1 . seldonio/seldon-core-s2i-python37-ubi8:${VERSION} seldonio/fixed-model:0.1
	s2i build -E environment_v2 . seldonio/seldon-core-s2i-python37-ubi8:${VERSION}  seldonio/fixed-model:0.2
```

so not yet affected by current changes.

**Comment by axsaucedo** [comment]
Nice one - looks good
/approve

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[agrski](https://github.com/SeldonIO/seldon-core/pull/3591#pullrequestreview-755311364 "Approved")*, *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3591#issuecomment-922816250 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
